spec/models/location_spec.rb

```diff
@@ -18,4 +18,13 @@
     described_class.create!(location_attrs)
     expect { described_class.create!(location_attrs) }.to raise_error(ActiveRecord::RecordInvalid)
   end
+
+  it "validates size of state abbreviation" do
+    location_attrs = { address: "123 Fake St.",
+                       city: "Bannock",
+                       state: "Idaho",
```

Comment:
Holy shit.  Good catch.  Totally missing `described_class.new(location_attrs)`.

Fixing and then merging.